cinema/views.py

```diff
@@ -140,9 +160,26 @@
         )
     )
     serializer_class = MovieSessionSerializer
-    authentication_classes = (TokenAuthentication,)
     permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "title",
+                type=OpenApiTypes.DATE,
+                description="Filter by movie session date year-month-day (e.g. &date=2024-12-31)", # noqa
```

Comment:
The description for the 'genres' parameter in the `extend_schema` decorator seems incorrect. It should describe filtering by genre IDs, not movie IDs. Ensure the description matches the actual filter functionality.